regulations/static/regulations/js/unittests/specs/views/comment-view-spec.js

```diff
@@ -0,0 +1,108 @@
+var chai = require('chai');
+var expect = chai.expect;
+var sinon = require('sinon');
+var jsdom = require('mocha-jsdom');
+var localStorage = require('node-localstorage');
+
+var storage = new localStorage.LocalStorage('.');
+
+describe('CommentView', function() {
+  jsdom();
+
+  var $el, commentView, CommentView, CommentEvents;
+
+  before(function() {
+    Backbone = require('backbone');
+    $ = require('jquery');
+    Backbone.$ = $;
+    CommentView = require('../../../source/views/comment-view');
+    CommentEvents = require('../../../source/events/comment-events');
+    window.localStorage = storage;
+  });
+
+  beforeEach(function() {
+    $el = $(
+      '<div class="comment">' +
+        '<div class="comment-clear">Clear</div>' +
```

Comment:
In #145 I change the position of where `comment-clear` class is located (below submit button)